server/routes/api/teas.js

```diff
@@ -73,7 +80,9 @@ router.delete("/delete-tea/:id", (req, res) => {
   });
 });
 
-// get the tea with that id (accessed at GET http://localhost:8080/api/teas/:tea_id)
+// @route GET api/teas/:id
+// @desc Get individual tea
+// @access Public
 router.get("/tea/:id", (req, res) => {
```

Comment:
The full path here is `/api/teas/tea/:id` - the `teas/tea` part seems redundant to me.